### PR TITLE
XS✔ ◾ ✨ New rule - POC and MVP

### PR DESCRIPTION
**Pull request by bradystroud**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

Email - RE: SSW Rule - POC vs MVP

I think its really important devs understand this so their clients can understand it

### CODE REVIEW

**Review by jaydenalchin — CHANGES_REQUESTED** [review]
@bradystroud great rule but make sure it's added to a category

**Comment by tiagov8** [comment]
> @bradystroud great rule but make sure it's added to a category

was about to say the same :)

File: rules/poc-vs-mvp/rule.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+type: rule
+archivedreason: 
+title: Do you understand the difference between POC and MVPs?
```
Comment (danielmackay) [inline rules/poc-vs-mvp/rule.md:4]: Either both terms should be singular or plural

```suggestion
title: Do you understand the difference between a POC and MVP?
```

**Review by danielmackay — DISMISSED** [review]
1 minor comment

**Comment by bradystroud** [comment]
@jaydenalchin 
> @bradystroud great rule but make sure it's added to a category

Done - Added a few

File: categories/communication/rules-to-better-communication.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ index:
   - corridor-conversations
   - tofu
   - attention-to-detail
+  - poc-vs-mvp
```
Comment (tiagov8) [inline categories/communication/rules-to-better-communication.md:30]: Hey @bradystroud 

Agreed this is important, but I don't think it belongs to "Communication" category

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: categories/communication/rules-to-better-communication.md (hunk 2 above)
Comment (bradystroud) [reply categories/communication/rules-to-better-communication.md]: I agree, removed

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**